packages/react-native-cli/src/commands/AutomateSymbolicationCommand.ts

```diff
@@ -38,15 +38,16 @@ export default async function run (projectRoot: string, urls: OnPremiseUrls): Pr
     const { androidIntegration } = await prompts({
       type: 'confirm',
       name: 'androidIntegration',
-      message: 'Do you want to automatically upload JavaScript source maps as part of the Gradle build?',
+      message: 'Do you want to install the BugSnag CLI to allow you to upload JavaScript source maps?',
       initial: true
     }, { onCancel })
 
     if (androidIntegration) {
       await enableReactNativeMappings(projectRoot, urls[UrlType.UPLOAD], urls[UrlType.BUILD], logger)
```

Comment:
In fact, I think we'd be better removing this task entirely as `uploadReactNativeMappings` defaults to false: there's no need to meddle with their Gradle files at all in a fresh project.

Better again would be to check for the presence of this setting and warn the user to disable it.

@lemnik – thoughts?

Reply:
Agreed in meeting to produce a warning here if `uploadReactNativeMappings` is found and set to `true` in the Gradle file.